Sources/Noora/NooraMock.swift

```diff
@@ -0,0 +1,53 @@
+#if DEBUG
+    import Rainbow
+
+    public class MockNoora: Noora, CustomStringConvertible {
```

Comment:
Given this is something we expect consumers of `Noora` to use to snapshot test the output, I'd add documentation along some usage guidance.